README.md

```diff
@@ -36,6 +36,14 @@ https://github.com/decidim/decidim/blob/master/docs/getting_started.md
 Then install PostGIS:
 https://postgis.net/install/
 
+### Setup (Docker)
+
+Run `bin/docker build` and then `bin/docker start`. You should be able to go to http://localhost:3000 when the process is complete. The initial build and first page load will take some time.
+
+The Dockerfile will create a postgres database (pg-data) and redis (redis-data) directories that will be persistent across containers.
+
+To enter the container in a bash environment, run `bin/docker bash`. To enter the postgres container using psql, run `bin/docker pg`
```

Comment:
There are lots of other Rails commands that devs will need access to, like `rails c` or `rails g controller`. I know `bin/docker bash` will open up the container that will let us run those commands, but should we add a way to run them from the outside-of-the-container command line? (See https://semaphoreci.com/community/tutorials/dockerizing-a-ruby-on-rails-application)